scripts/publish_sandbox.sh

```diff
@@ -10,7 +10,7 @@ if [ -z "$VERSION" ]; then
 fi
 
 ./scripts/build_layers.sh
-VERSION=$VERSION REGIONS=sa-east-1 ./scripts/publish_layers.sh
+VERSION=$VERSION REGIONS=us-east-1 ./scripts/publish_layers.sh
```

Comment:
should we keep this as `sa-east-1`?

Reply:
OH good question, this is intentional - our RC now runs in us-east-1 and production self monitoring runs in sa-east-1, so I think we want this change? Will ask Alex.